mentoring-wg/faqs.md

```diff
@@ -0,0 +1,33 @@
+CNCF TAG Contributor Strategy
+# Mentoring Working Group
+
+
+## About TAGCS Mentorship Working Group
+
+[Mentoring WG Charter](https://github.com/cncf/tag-contributor-strategy/tree/main/mentoring)
+
+* CNCF Slack: [#tag-contributor-strategy](https://cloud-native.slack.com/archives/CT6CWS1JN)
+* [Discussion boards](https://github.com/cncf/mentoring/discussions)
+* [Email list](https://lists.cncf.io/g/tag-cs-mentoring-wg/)
+
+
+## Meeting details
+
+### Recurring monthly
+* 2nd Tuesday of the month at 8PM UTC
+* 4th Tuesday of the month at 8PM UTC (during the setup phase)
+
+[CNCF Public Events - TAG CS Mentoring WG](https://tockify.com/cncf.public.events/monthly?search=CNCF%20TAG%20Contributor%20Strategy%20Mentoring%20WG)
+
+### Zoom
+
+Zoom Meeting  
+https://zoom.us/my/cncftagcontributorstrategy?pwd=TnI0WU9Eb2I1RlRWdkl1R0k1WkZXUT09
+
+Passcode: 77777
+
+# Meeting Minutes
+
+* [2022](./2022-meeting-minutes.md) (current)
```

Comment:
Now that we have a 2023 page for minutes, we should link to it here

Reply:
+1

Reply:
We removed mentoring-wg faqs — i need to double check that we have the 2023 meeting minutes in the main mentoring-wg readme